Index: Jenkinsfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>pipeline {\r\n  agent any\r\n  stages {\r\n    stage ('Build') {\r\n      steps {\r\n        echo 'Running build automation'\r\n        sh './gradlew build --no-daemon'\r\n        archiveArtifacts artifacts: 'react.zip'\r\n      }\r\n    }\r\n  }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Jenkinsfile b/Jenkinsfile
--- a/Jenkinsfile	(revision b6826433b67526c713803be2cdcadd2238b59ce7)
+++ b/Jenkinsfile	(date 1650449142526)
@@ -1,12 +1,11 @@
 pipeline {
-  agent any
-  stages {
-    stage ('Build') {
-      steps {
-        echo 'Running build automation'
-        sh './gradlew build --no-daemon'
-        archiveArtifacts artifacts: 'react.zip'
-      }
-    }
-  }
+    agent any
+
+    stages {
+        stage('Hello') {
+            steps {
+                echo 'Hello World'
+            }
+        }
+    }
 }
